searchtweets/result_stream.py

```diff
@@ -27,6 +27,10 @@
 
 logger = logging.getLogger(__name__)
 
+class NoAuth(requests.auth.AuthBase):
```

Comment:
It would be nice if requests library already had something like this so wouldn't have to extend it like this here, but looks like requests.auth doesn't have this

Reply:
good night, error persists...